int16 classes
float32 posX
float32 posY
float32 posZ
float32 size
float32 distance
float32 x_max
float32 x_min
float32 y_max
float32 y_min
float32 z_max